src/components/MobileGridView/MobileGridView.tsx

```diff
@@ -1,8 +1,8 @@
 import { CSSProperties } from 'react';
-import { makeStyles, createStyles, Theme } from '@material-ui/core';
+import { makeStyles, createStyles, Theme, useTheme, useMediaQuery } from '@material-ui/core';
```

Comment:
I hate to comment on code that's not part of a PR, but I was just thinking about the pagination indicators. What if we only adjust the layout to leave space for them only when they are present? Like this:

![page-indicators](https://user-images.githubusercontent.com/12685223/174186374-4a9ae188-574d-474b-a83f-2ef9fb338038.gif)

This CSS seems to do the trick:
```
const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    participantContainer: {
      background: theme.gridViewBackgroundColor,
      position: 'absolute',
      top: 0,
      right: 0,
      bottom: 0,
      left: 0,
      height: '100%',
      '& .swiper': {
        height: '100%',
        '--swiper-pagination-bullet-inactive-color': 'white',
      },
      '& .swiper-wrapper': {
        height: '100%',
      },
      '& .swiper-pagination.swiper-pagination-bullets': {
        bottom: '5px',
      },
    },
    isPaginationActive: {
      '& .swiper-slide': { // To leave room for the pagination indicators
        height: 'calc(100% - 21px)', 
        paddingBottom: '21px',
      },
    },
    swiperSlide: {
      display: 'flex',
      flexWrap: 'wrap',
      alignSelf: 'center',
      alignContent: 'flex-start',
    },
  })
);
```

and then with clsx:
```
<div className={clsx(classes.participantContainer, { [classes.isPaginationActive]: remoteParticipantCount > 5 })}>
// ...
```

What do you think?

Reply:
Awesome that looks great